userspace/falco/app/actions/init_falco_engine.cpp

```diff
@@ -32,6 +66,17 @@ void configure_output_format(falco::app::state& s) {
 			                                  false);
 		}
 
+		// Add suggested filtercheck formats to each source output
+		if(eo.m_suggested_output) {
```

Comment:
If `suggested_output` is enabled:
* if any matching source is specified, we only list fields for that source, and add the extra output format for it
* else, we add the extra output format for all loaded sources

`Tags` and `Rule` are still managed normally.

Reply:
Why does this need special treatment? If you don't specify any source the extra format already applies to all sources.

Reply:
... it needs special treatment because every source has its own fields and thus its own format so in order to select the correct one we need to iterate through all the sources. Makes sense. Thanks @FedeDP